docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -7,14 +7,23 @@ external `PostgreSQL <https://www.postgresql.org/>`_ database. This can be used
 different systems like PostgreSQL and Hive, or between different
 PostgreSQL instances.
 
+Requirements
+------------
+
+The connector supports PostgreSQL 9.6 and higher, as well as compatible
+commercial PostgreSQL distributions and derivative databases compatible with the
+PostgreSQL JDBC driver, as connected data sources.
```

Comment:
Define "compatible".

We don't test and have no plans to test with any commercial PostgreSQL distributions and derivative databases.
As long as it works, it works, but it's not "supported".

> databases compatible with the PostgreSQL JDBC driver

One can argue Redshift is such database, or Greenplum (or perhaps many others), but they are _not_ "supported" when used with this connector.


Remove all the "as well as compatible
commercial PostgreSQL distributions and derivative databases compatible with the
PostgreSQL JDBC driver"

Reply:
> since we dont really support anything (since we are an open source project).

doesn't mean we should never use the word "supported"
i think we can. People come asking for help and we try to help them. People find bugs and we decide whether a thing is a bug or not. This is part of definition of "supported" to me.

but, we should use the word "supported" reasonably -- eg we should not claim support for things that just "may happen to work" but we wouldn't spend time fixing them if we do not